monai/transforms/croppad/dictionary.py

```diff
@@ -862,18 +864,16 @@ def __call__(self, data: Mapping[Hashable, torch.Tensor]) -> List[Dict[Hashable,
         # initialize returned list with shallow copy to preserve key ordering
         ret: List = [{} for _ in range(self.cropper.num_samples)]
         # deep copy all the unmodified data
-        for key in set(data.keys()).difference(set(self.keys)):
+        for key in set(d.keys()).difference(set(self.keys)):
```

Comment:
It seems there is some reason to use `data` instead of `d` here, but I really can't remember it.. Let me try to recall.

Thanks.

Reply:
sure, I wait for your recall.

Reply:
It seems in the previous python versions `d = dict(data)` can't ensure the items order in `d`, anyway, we don't have this issue now and I see both `CropByPosNeg` and `CropByClasses` are already changed to use `d` instead.

Thanks.